client/src/rpc_response.rs

```diff
@@ -362,6 +362,7 @@ pub struct RpcPerfSample {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct RpcInflationReward {
+    pub address: String,
```

Comment:
Is this needed?  Instead return `RpcInflationReward`s in the same order as the request?

Reply:
Yeah, we can do that instead.

Reply:
A queried address could be absent from `block.rewards`, right? I'm not sure an `RpcInflationReward { amount: 0, .. }` makes sense in these cases, since we don't know `post_balance`

Reply:
`Vec<Option<RpcInflationReward>>` as a return type?

Reply:
> `Vec<Option<RpcInflationReward>>` as a return type?

Wfm!

Reply:
Yep, that's what I went with!